packages/design-system-tokens/src/tokens/font.ts

```diff
@@ -4,10 +4,10 @@
 import { FontTokens, to } from '../lib/types';
 
 const font = to<FontTokens>()({
-  'family-open-sans': "'Open Sans', Helvetica, sans-serif",
-  'family-rubik': "'Rubik', sans-serif",
-  'family-montserrat': "'Montserrat', sans-serif",
-  'family-bitter': 'Bitter, Georgia, serif',
+  'family-open-sans': "'Open Sans', Inter, Roboto, 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', Arial, sans-serif",
```

Comment:
![neo-grotesque](https://github.com/CMSgov/design-system/assets/5159392/f0859b67-31fe-4a0d-a458-f769ceddfbfa)

Reply:
since one of the fallbacks defined was helvetica, this option seemed most appropriate